### PR TITLE
doc/dev/continuous-integration: document chacra

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by ideepika** [comment]
jenkins render docs

**Comment by ideepika** [comment]
@tchaikov unrelated render docs failure: 
```
./admin/build-doc: missing required packages, please install them:
sudo apt-get install -o APT::Install-Recommends=true python3-venv
```

**Comment by tchaikov** [comment]
> @tchaikov unrelated render docs failure:
> 
> ```
> ./admin/build-doc: missing required packages, please install them:
> sudo apt-get install -o APT::Install-Recommends=true python3-venv
> ```
> 
> let me know if I can ou help out

could you please work with David on understand how to install `python3-venv` on `docs.ceph.com`? thought it was addressed by https://github.com/ceph/ceph-cm-ansible/pull/643 ..

**Comment by djgalloway** [comment]
> could you please work with David on installing `python3-venv` on `docs.ceph.com`? thought it was addressed by [ceph/ceph-cm-ansible#643](https://github.com/ceph/ceph-cm-ansible/pull/643) ..

Done

File: doc/dev/continuous-integration.rst [hunk 1]
```diff
@@ -91,11 +91,40 @@ Chacra
 Shaman
    is a server offering RESTful API allowing the clients to query the
    information of repos hosted by chacra nodes. Shaman is also known
-   for its `Web UI`_. But please note, shaman does not build the
+   for its `Shaman`_. But please note, shaman does not build the
```
Comment (djgalloway) [inline doc/dev/continuous-integration.rst:94]: Shaman is also known for its Shaman?

**Review by jdurgin — APPROVED** [review]
looks good, just a couple small things

File: doc/dev/continuous-integration.rst (hunk 1 above)
Comment (tchaikov) [reply doc/dev/continuous-integration.rst]: sorry, thought i could reuse this link.

File: doc/dev/continuous-integration.rst [hunk 2]
```diff
@@ -166,3 +205,68 @@ we need to upgrade these third party libraries, we should
...
+
+distro
+    the distro name for which the packages are built. Currently, following distros are
+    supported:
+
+    - centos
+    - debian
+    - fedora
+    - rhel
+    - ubunptu
```
[the remark this answers is not in the document]
Comment (tchaikov) [reply doc/dev/continuous-integration.rst]: fixed.

File: doc/dev/continuous-integration.rst [hunk 3]
```diff
@@ -166,3 +205,68 @@ we need to upgrade these third party libraries, we should
...
+project
+    in general, it is used for denoting a set of related packages. For instance,
+    ``libboost``.
+
+branch
+    branch of project. This mirrors the concept of a Git repo.
+
+ref
+    a unique id of a given version of a set packages. It is quite like the version of
+    the packages as a whole. It is a good practice to version the packaging receipe,
```
[the remark this answers is not in the document]
Comment (tchaikov) [reply doc/dev/continuous-integration.rst]: sorry for the typo. fixed and repushed.

**Pull request merged**